src/Tutor.Core/DomainModel/AssessmentEvents/AssessmentEventInteraction.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Tutor.Core.DomainModel.AssessmentEvents
+{
+    public abstract class AssessmentEventInteraction
```

Comment:
I feel like this name is too generic, but I couldn't think of a better one. We need something that is still generic, but not too much or too little.